mev_inspect/transfers.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict, List, Optional
+
+from mev_inspect.schemas.classified_traces import Classification, ClassifiedTrace
+from mev_inspect.schemas.transfers import Transfer
+from mev_inspect.traces import is_child_trace_address, get_child_traces
+
+
+def get_child_transfers(
+    transaction_hash: str,
+    parent_trace_address: List[int],
+    traces: List[ClassifiedTrace],
+) -> List[Transfer]:
+    child_transfers = []
+
+    for child_trace in get_child_traces(transaction_hash, parent_trace_address, traces):
+        if child_trace.classification == Classification.transfer:
+            child_transfers.append(Transfer.from_trace(child_trace))
+
+    return child_transfers
+
+
+def filter_transfers(
+    transfers: List[Transfer],
+    to_address: Optional[str] = None,
+    from_address: Optional[str] = None,
+) -> List[Transfer]:
+    filtered_transfers = []
+
+    for transfer in transfers:
+        if to_address is not None and transfer.to_address != to_address:
+            continue
+
+        if from_address is not None and transfer.from_address != from_address:
+            continue
+
+        filtered_transfers.append(transfer)
+
+    return filtered_transfers
+
+
+def remove_inner_transfers(transfers: List[Transfer]) -> List[Transfer]:
```

Comment:
Can you share a bit about `inner_transfers` int his context? I see usage in `_get_swaps_for_transaction` and `test_remove_inner_transfers` (which returns as expected) but still not sure I follow what's going on here wrt swaps.

Reply:
This is a good call out. I was struggling to find a good name for this

Basically I noticed that USDC specifically internally has another ERC-20 that it calls (FiatToken iirc). So you'll see an ERC-20 transfer with a child trace of another ERC-20 transfer of all the same parameters but a separate token address.

For the swaps, it's easier to ignore that implementation detail and just say that we swapped the parent token (USDC). So this removed any transfers which are children of other transfers

Maybe `remove_child_transfers_of_transfers`?

Reply:
Makes sense, better renaming (or just adding code comments for context around USDC-like tokens) should be good. The rest of the PR makes sense too so should be good to merge post this fix.